packages/account/src/Components/address-details/address-details.tsx

```diff
@@ -285,7 +276,11 @@ const AddressDetails = ({
                                                                 placeholder={localize('Please select')}
                                                                 label={localize('State/Province')}
                                                                 value={address_state_to_display || values.address_state}
-                                                                list_items={states_list}
+                                                                list_items={
+                                                                    states_list as React.ComponentProps<
+                                                                        typeof SelectNative
+                                                                    >['list_items']
+                                                                }
```

Comment:
Instead of Type assertion, can we try modifying the `states_list` in a way this component expects?
You can ignore this comment as well

Reply:
Actually, I also don’t like this approach. But we can’t modify states_list, we should modify 'SelectNative' components types.
Will check it again. thank you